remme/rest_api/certificate_api_decorator.py

```diff
@@ -75,10 +76,28 @@ def validation_logic(payload):
                                                  default_backend())
             if not is_valid_token_balance():
                 return {'error': 'You have no tokens to issue certificate'}, 402
+
+            not_valid_before = payload.get('not_valid_before', None)
+            not_valid_after = payload.get('not_valid_after', None)
```

Comment:
Don't we want to make not_valid_before - optional ? 
if user provide just not_valid_after - do we want to use current date as a start date ?

Reply:
Essentially this way we supposed it to work.

Reply:
Agreed, polished it